NaviGator/utils/voltage_gui/src/voltage_gui.py

```diff
@@ -80,27 +107,88 @@ def __init__(self):
 
     # The functions that the subscribers call in order to get new data
     def updateMain(self, mainData: Float32) -> None:
+        """
+        It is one of the functions that the subscribers call in order to get new data.
+
+        Args:
+            mainData: it is a float value that is being passed in for a variable to be set to.
+
+        Returns:
+            No statement is being returned.  
+        """
         self.battery_voltage = mainData
 
     def update_FL(self, dataFL: Feedback) -> None:
+        """
+        It is one of the functions that the subscribers call in order to get new data.
+
+        Args:
+            dataFL: it is a feedback value that is being passed in for a variable to be set to.
+
+        Returns:
+            No statement is being returned.  
+        """
         self.voltageFL = dataFL.supply_voltage
 
     def update_FR(self, dataFR: Feedback) -> None:
+        """
+        It is one of the functions that the subscribers call in order to get new data.
+
+        Args:
+            dataFR: it is a feedback value that is being passed in for a variable to be set to.
+
+        Returns:
+            No statement is being returned.  
+        """
         self.voltageFR = dataFR.supply_voltage
 
     def update_BL(self, dataBL: Feedback) -> None:
+        """
+        It is one of the functions that the subscribers call in order to get new data.
+
+        Args:
+            dataBL: it is a feedback value that is being passed in for a variable to be set to.
+
+        Returns:
+            No statement is being returned.  
+        """
         self.voltageBL = dataBL.supply_voltage
 
     def update_BR(self, dataBR: Feedback) -> None:
+        """
+        It is one of the functions that the subscribers call in order to get new data.
+
+        Args:
+            dataBR: it is a feedback value that is being passed in for a variable to be set to.
+
+        Returns:
+            No statement is being returned.  
+        """
         self.voltageBR = dataBR.supply_voltage
 
-    # Part of signal that notifies program whenever window is resized
-    def resizeEvent(self, event):
+    def resizeEvent(self, event): #done
+        """
+        Part of signal that notifies program whenever window is resized
+
+        Args:
+            event: an event object is being passed in.
+
+        Returns:
+            An event was being returned with a different size. 
+        """
         self.resized.emit()
         return super().resizeEvent(event)
 
-    # Increase/decrease size of fonts based on window resize
-    def resizeFont(self) -> None:
+    def resizeFont(self) -> None: #done
+        """
+        Increase/decrease size of fonts based on window resize
+
+        Args:
+            No arguments are being passed in. 
+
```

Comment:
Same for the args: for functions that take in no arguments, we can just omit this section from the doc of the function!